samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/ManyGraphics.razor

```diff
@@ -21,8 +21,8 @@
         <SpatialReference Wkid="102100" />
     </Extent>
     <Map>
-        <OpenStreetMapLayer />
         <GraphicsLayer @ref="_graphicsLayer" />
+        <OpenStreetMapLayer />
```

Comment:
I did not see anything here or in `Featurelayers` dont know if maybe it would be a todo or some information in the GB documentation to link to the ESRI docs, but an explanation of layer/data ordering to affect the display of those different items that would be in the operational layer.

Reply:
It is in the release notes, first item under `Breaking Changes`, but I agree, we should probably have a dedicated docs page.